src/compile/steps/write-to-note.ts

```diff
@@ -44,22 +107,30 @@ export const WriteToNoteStep = makeBuiltinStep({
       if (!target || target.length == 0) {
         throw new Error("Invalid path for Save as Note.");
       }
-      const file = target.endsWith(".md") ? target : target + ".md";
-      const path = normalizePath(`${context.projectPath}/${file}`);
 
-      const pathComponents = path.split("/");
-      pathComponents.pop();
+      const filePath = resolvePath(context.projectPath, target);
+      const conatiningFolderParts = filePath.split("/");
+      conatiningFolderParts.pop();
+      const containingFolderPath = conatiningFolderParts.join("/");
 
       try {
-        await context.app.vault.createFolder(pathComponents.join("/"));
+        await context.app.vault.createFolder(containingFolderPath);
       } catch (e) {
         // do nothing, folder already existed
       }
 
-      await context.app.vault.adapter.write(path, input.contents);
+      console.log('Writing to:', filePath);
```

Comment:
I've tried to namespace all logged text with `[Longform]`:

```suggestion
      console.log('[Longform] Writing to:', filePath);
```

Reply:
Good to know!  I'll keep that in mind for future logging stuff 🙂